pkgs/tools/networking/network-manager/default.nix

```
@@ -94,7 +94,7 @@ in stdenv.mkDerivation rec {
   doCheck = false; # requires /sys, the net
 
   preInstall = ''
-    installFlagsArray=( "sysconfdir=$out/etc" "localstatedir=$out/var" "runstatedir=$out/var/run" )
+    installFlagsArray=( "sysconfdir=$out/etc" "localstatedir=$out/var" "runstatedir=/run" )
```

This probably should be:
```suggestion
    installFlagsArray=( "sysconfdir=$out/etc" "localstatedir=$out/var" "runstatedir=$out/run" )
```

---

Keeping the old code in mind, your comment makes sense. However, I do wonder whether `runstatedir` should just be `/run` instead of `$out/run`.

---

Judging from the `installFlagsArray` it's so things get installed to $out/run and not /run which is outside the build prefix.